### ISSUE TITLE
Hide other people's votes in voting phase

### ISSUE THREAD

**Issue by heliosam** [retro-tool/retro-tool#17]
Seeing other team member's votes may bias your own votes

**Comment by uesteibar** [comment]
To make it clear, I understand the acceptance criteria here would be that:

- In the voting phase, you only see your own votes.
- In the following phases, all votes are counted.

Is that right? In that case, this can be done with a BE change.

**Comment by heliosam** [comment]
Yep, that would be it

**Comment by uesteibar** [comment]
Sounds good to me but let's run it by the rest of team, what do you think @retro-tool/all ?

**Comment by atrzaska** [comment]
👍

**Comment by marciobarrios** [comment]
I agree, for me is not too important but it's not the first feedback that we got asking for it. Let's do it!

**Comment by rudeayelo** [comment]
👍
